lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -31,6 +31,118 @@
+   *       Classification}, else unchanged
+   *   <li><b>{@linkplain Task#getDescription() description}</b> - if NULL then description of its
+   *       {@linkplain Classification}, else unchanged
+   *   <li><b>{@linkplain Task#getCreator() creator}</b> - id of current user
+   *   <li><b>{@linkplain Task#getState() state}</b> - {@linkplain TaskState#READY}
+   *   <li><b>{@linkplain Task#isRead() isRead}</b> - false
+   *   <li><b>{@linkplain Task#isTransferred() isTransferred}</b> - false
+   * </ul>
+   *
+   * @param taskToCreate the transient {@linkplain Task} to be inserted
+   * @return the created and inserted {@linkplain Task}
+   * @throws TaskAlreadyExistException if the {@linkplain Task} already exists
+   * @throws NotAuthorizedException if the current user has no {@linkplain
+   *     WorkbasketPermission#APPEND} for the {@linkplain Workbasket} the {@linkplain Task} is in
+   * @throws WorkbasketNotFoundException if the {@linkplain Workbasket} referenced by the
+   *     {@linkplain Task#getWorkbasketSummary() workbasketSummary} of the {@linkplain Task} isn't
+   *     found
+   * @throws ClassificationNotFoundException if the {@linkplain Classification} referenced by
+   *     {@linkplain Task#getClassificationSummary() classificationSummary} of the {@linkplain Task}
+   *     isn't found
+   * @throws InvalidArgumentException if the {@linkplain Task#getPrimaryObjRef() primaryObjRef} is
+   *     invalid
+   * @throws AttachmentPersistenceException if an {@linkplain Attachment} with the same {@linkplain
+   *     Attachment#getId() id} was added to the {@linkplain Task} multiple times without using
+   *     {@linkplain Task#addAttachment(Attachment)}
+   * @throws ObjectReferencePersistenceException if an {@linkplain ObjectReference} with the same
+   *     {@linkplain ObjectReference#getId() id} was added to the {@linkplain Task} multiple times
+   *     without using {@linkplain Task#addSecondaryObjectReference(ObjectReference)}
+   */
+  Task createTask(Task taskToCreate)
+      throws NotAuthorizedException, WorkbasketNotFoundException, ClassificationNotFoundException,
+          TaskAlreadyExistException, InvalidArgumentException, AttachmentPersistenceException,
+          ObjectReferencePersistenceException;
+
+  // READ
+
+  /**
+   * Fetches a {@linkplain Task} from the database by the specified {@linkplain Task#getId() id}.
+   *
+   * @param taskId the {@linkplain Task#getId() id} of the {@linkplain Task}
+   * @return the {@linkplain Task} with the specified taskId
+   * @throws TaskNotFoundException if the {@linkplain Task} with taskId wasn't found
+   * @throws NotAuthorizedException if the current user has no {@linkplain
+   *     WorkbasketPermission#READ} for the {@linkplain Workbasket} the {@linkplain Task} is in
+   */
+  Task getTask(String taskId) throws TaskNotFoundException, NotAuthorizedException;
+
+  // UPDATE
```

Comment:
Why do you have this comment here? Neither the methods above nor below are updating the Task